dbld/rpm

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+cd /source
+if [ ! -f syslog-ng.spec ]; then
+	echo "You don't seem to have an RPM specfile in the root of your source tree, this means one of the followings:"
+	echo "  1) you don't do this from a release tarball"
+	echo "  2) you haven't merged the 'release' branch"
+	echo "  3) you didn't create rpm packaging manually"
+	echo ""
+	echo "Please make sure you have obtained a source distribution with a spec file and rerun this command"
+	exit 1
+fi
+VERSION=`cat VERSION`
+
+cd /build
+rpmbuild -ta syslog-ng-${VERSION}.tar.gz
```

Comment:
I guess we forgot to copy the generated rpm-s into the /build directory. 
The rpmbuild command ended with this lines: 
```
Checking for unpackaged file(s): /usr/lib/rpm/check-files /home/micek/rpmbuild/BUILDROOT/syslog-ng-3.9.1-20.el7.centos.x86_64
Wrote: /home/micek/rpmbuild/SRPMS/syslog-ng-3.9.1-20.el7.centos.src.rpm
Wrote: /home/micek/rpmbuild/RPMS/x86_64/syslog-ng-3.9.1-20.el7.centos.x86_64.rpm
Wrote: /home/micek/rpmbuild/RPMS/x86_64/syslog-ng-libdbi-3.9.1-20.el7.centos.x86_64.rpm
Wrote: /home/micek/rpmbuild/RPMS/x86_64/syslog-ng-mongodb-3.9.1-20.el7.centos.x86_64.rpm
Wrote: /home/micek/rpmbuild/RPMS/x86_64/syslog-ng-smtp-3.9.1-20.el7.centos.x86_64.rpm
Wrote: /home/micek/rpmbuild/RPMS/x86_64/syslog-ng-java-3.9.1-20.el7.centos.x86_64.rpm
Wrote: /home/micek/rpmbuild/RPMS/x86_64/syslog-ng-geoip-3.9.1-20.el7.centos.x86_64.rpm
Wrote: /home/micek/rpmbuild/RPMS/x86_64/syslog-ng-redis-3.9.1-20.el7.centos.x86_64.rpm
Wrote: /home/micek/rpmbuild/RPMS/x86_64/syslog-ng-riemann-3.9.1-20.el7.centos.x86_64.rpm
Wrote: /home/micek/rpmbuild/RPMS/x86_64/syslog-ng-http-3.9.1-20.el7.centos.x86_64.rpm
Wrote: /home/micek/rpmbuild/RPMS/x86_64/syslog-ng-python-3.9.1-20.el7.centos.x86_64.rpm
Wrote: /home/micek/rpmbuild/RPMS/x86_64/syslog-ng-devel-3.9.1-20.el7.centos.x86_64.rpm
Wrote: /home/micek/rpmbuild/RPMS/x86_64/syslog-ng-debuginfo-3.9.1-20.el7.centos.x86_64.rpm
Executing(%clean): /bin/sh -e /var/tmp/rpm-tmp.2ICEHz
+ umask 022
+ cd /home/micek/rpmbuild/BUILD
+ cd syslog-ng-3.9.1
+ /usr/bin/rm -rf /home/micek/rpmbuild/BUILDROOT/syslog-ng-3.9.1-20.el7.centos.x86_64
+ exit 0
Your rpms are in /build, also available on the host in $(top_srcdir)/dlbd/build
```
The rpm packages are under: `/home/micek/rpmbuild/RPMS/x86_64/`.